docs/advance-toolchain-documentation.html

```diff
@@ -89,7 +90,8 @@
 <ul>
   <li>PPC970, POWER4, POWER5, POWER5+, POWER6, POWER6x and POWER7 optimized runtime libraries</li>
 </ul>
-<p>This self-contained toolchain does not rely on the system toolchain and requires minimum dependencies. Nor does it override the default Linux distribution toolchain (it is installed in <span style="font-family:courier new,courier,monospace;">/opt</span>). The latest release includes current stable versions of the following packages:</p>
+<p>This self-contained toolchain does not rely on the system toolchain and requires minimum dependencies. Nor does it override the default Linux distribution toolchain (it is installed in <span style="font-family:courier new,courier,monospace;">/opt</span>). The Advance Toolchain source code can be found at: <a href="https://github.com/advancetoolchain/advance-toolchain">https://github.com/advancetoolchain/advance-toolchain</a>.</p>
```

Comment:
Let's add a paragraph break between the two "self-contained" sentences and the "source code" sentences, as these are independent thoughts.